index.html

```diff
@@ -54,8 +54,181 @@ <h2>I'm a software developer</h2>
+      </div>
+    </section>
+    <section class="my-work-detail">
+      <div class="grid-container">
+        <div class="my-work-card">
+          <div class="card-image"></div>
+          <div class="my-works-text">
+            <h2>Multi-post Stories Gain+Glory</h2>
+            <ul class="built-with">
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>Jav Script</li>
+              <li>html</li>
+            </ul>
+            <input type="button" class="work-button" value="See Project">
+          </div>
+        </div>
+        <div class="my-work-card">
+          <div class="card-image"></div>
+          <div class="my-works-text">
+            <h2>Multi-post Stories Gain+Glory</h2>
+            <ul class="built-with">
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>Jav Script</li>
+              <li>html</li>
+            </ul>
+            <input type="button" class="work-button" value="See Project">
+          </div>
+        </div>
+        <div class="my-work-card">
+          <div class="card-image"></div>
+          <div class="my-works-text">
+            <h2>Multi-post Stories Gain+Glory</h2>
+            <ul class="built-with">
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>Jav Script</li>
+              <li>html</li>
+            </ul>
+            <input type="button" class="work-button" value="See Project">
+          </div>
+        </div>
+        <div class="my-work-card">
+          <div class="card-image"></div>
+          <div class="my-works-text">
+            <h2>Multi-post Stories Gain+Glory</h2>
+            <ul class="built-with">
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>Jav Script</li>
+              <li>html</li>
+            </ul>
+            <input type="button" class="work-button" value="See Project">
+          </div>
+        </div>
+        <div class="my-work-card">
+          <div class="card-image"></div>
+          <div class="my-works-text">
+            <h2>Multi-post Stories Gain+Glory</h2>
+            <ul class="built-with">
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>Jav Script</li>
+              <li>html</li>
+            </ul>
+            <input type="button" class="work-button" value="See Project">
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="about-me-section">
+      <div class="about-me">
+        <h2>About me</h2>
+        <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+          don’t hestiate to contact me.</p>
+        <input type="button" class="about-me-button" value="Get My Resume">
+      </div>
+    </section>
+    <section class="skill-section">
+      <div class="grid-container2">
+        <div class="my-work-card2">
```

Comment:
- [ ] Please, apply the same section advice for the about `section` too. You should wrap the title and the content in the same section.